oracle/price-feeder/README.md

```diff
@@ -9,6 +9,66 @@ This is a standalone version of [Umee's fantastic work](https://github.com/umee-
 
 ---
 
+
+# Setup
+If a cluster is running Oracle price-feeder, your validator is also required to run a price feeder or your validator will be jailed for missing votes.
+
+
+## Create an account for Oracle Price Feeder Delegate
+1) To avoid account sequence errors, it's reccomended to create a different account as an Oracle delegate. To do so, you'll need to create the account with
```

Comment:
Nit: this is to avoid messing up the account sequence number with the existing account. But we can't avoid sequence mismatch error.